### DISCUSSION TITLE
How do I ser number of replicas with HELM? I want to distribute database across multiple nodes, because I don't have enough memory on a single node.

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#22891] [Q&A and General discussion]
How do I ser number of replicas with HELM? I want to distribute database across multiple nodes, because I don't have enough memory on a single node.

**Comment by yhmo** [comment]
Milvus can manage replicas at collection level. It balances segments across all the query nodes.
https://milvus.io/docs/replica.md#In-Memory-Replica

You can set "replica_number" in the load() API:
```
collection = Collection(name="xxx")
collection.load(replica_number=2)
```

**Reply by martinenkoEduard** [reply]
But will it balance data across query nodes if replica_number=2 set to 1?

**Reply by yhmo** [reply]
> But will it balance data across query nodes if replica_number=2 set to 1?

Segments also can be balanced if you set replica_num=1
